tests/acceptance/test.spec.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs = require('fs-extra');
+var ng = require('../helpers/ng');
+var existsSync = require('exists-sync');
+var expect = require('chai').expect;
+var forEach = require('lodash/collection/forEach');
+var walkSync = require('walk-sync');
+var Blueprint = require('ember-cli/lib/models/blueprint');
+var path = require('path');
+var tmp = require('../helpers/tmp');
+var root = process.cwd();
+var util = require('util');
+var conf = require('ember-cli/tests/helpers/conf');
+var EOL = require('os').EOL;
+
+describe('Acceptance: ng test', function () {
+  before(conf.setup);
+
+  after(conf.restore);
+
+  beforeEach(function () {
+    return tmp.setup('./tmp')
+      .then(function () {
+        process.chdir('./tmp');
+      })
+      .then(function () {
+        return ng([
+          'init',
+          '--skip-npm',
+          '--skip-bower'
+        ]);
+      });
+  });
+
+  afterEach(function () {
+    this.timeout(10000);
+
+    return tmp.teardown('./tmp');
+  });
+
+  it('ng test should excecute a test', function () {
+    console.log('starting test');
+    return ng([
+      'test',
+      '--skip-npm',
+      '--skip-bower'
+    ])
+      .then(function () {
+        console.log('test completed');
+      });
```

Comment:
The addition of the `--single-run` command line arg will handle the karma server shut down. I've moved the karma server creation into the run function (no longer in init) to allow for passing of command line args it has to be in `run`